packages/intrepid2/src/Projection/Intrepid2_ProjectionStruct.hpp

```diff
@@ -90,16 +90,19 @@ class ProjectionStruct {
   enum EvalPointsType {BASIS, TARGET};
 
   typedef Kokkos::pair<ordinal_type,ordinal_type> range_type;
-  typedef typename Kokkos::Impl::is_space<DeviceType>::host_mirror_space::execution_space HostSpaceType;
-  typedef Kokkos::DynRankView<ValueType,HostSpaceType> view_type;
-  typedef Kokkos::View<range_type**,HostSpaceType> range_tag;
+  /// KK : do we really need this complication instead of using default host exec space ????
```

Comment:
I don't think we do. I think we can use the default memory space.

Reply:
@CamelliaDPG Is it fine that I replace  is_space<>::host_execution_space with default host exec/memory space ?

Reply:
@kyungjoo-kim, yes, I agree.